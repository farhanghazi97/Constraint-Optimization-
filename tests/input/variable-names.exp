m1:mon 9am
meeting2:mon 9am
meeting-3:mon 9am
meetingfour:mon 9am
meeting_five:mon 9am
6meeting:mon 9am
7_meeting-about-stuff:mon 9am
a-really-really-long-meeting-about-the-number-eight:mon 9am
EPICMEETING_NUMBER9:mon 9am
MeetingAbout9Asterisks*:mon 9am
$doll@rm1te`Meeting!:mon 9am
cost:0
