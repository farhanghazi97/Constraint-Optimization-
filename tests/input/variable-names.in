# Test variable name formats
meeting, m1
meeting, meeting2
meeting, meeting-3
meeting, meetingfour
meeting, meeting_five
meeting, 6meeting
meeting, 7_meeting-about-stuff
meeting, a-really-really-long-meeting-about-the-number-eight
meeting, EPICMEETING_NUMBER9
meeting, MeetingAbout9Asterisks*
meeting, $doll@rm1te`Meeting!
