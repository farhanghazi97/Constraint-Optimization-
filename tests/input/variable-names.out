m1:wed 1pm
meeting2:wed 1pm
meeting-3:wed 1pm
meetingfour:wed 1pm
meeting_five:wed 1pm
6meeting:wed 1pm
7_meeting-about-stuff:wed 1pm
a-really-really-long-meeting-about-the-number-eight:wed 1pm
EPICMEETING_NUMBER9:wed 1pm
MeetingAbout9Asterisks*:wed 1pm
$doll@rm1te`Meeting!:wed 1pm
cost:0
